.github/workflows/release-please.yml

```diff
@@ -21,13 +21,7 @@ jobs:
             [
               { "type": "feat", "section": "Features", "hidden": false },
               { "type": "fix", "section": "Bug Fixes", "hidden": false },
-              { "type": "docs", "section": "Documentation", "hidden": false },
-              { "type": "build", "section": "Build Related", "hidden": false },
-              { "type": "chore", "section": "Chores", "hidden": false },
-              { "type": "perf", "section": "Chores", "hidden": false },
-              { "type": "ci", "section": "Chores", "hidden": false },
-              { "type": "refactor", "section": "Chores", "hidden": false },
-              { "type": "test", "section": "Chores", "hidden": false }
+              { "type": "perf", "section": "Chores", "hidden": false }
```

Comment:
Should we maybe rename this section to "Performance Improvements"?

Reply:
Renamed in https://github.com/eslint/eslint-visitor-keys/pull/68/commits/a1722cf7a3f93795abcfad4a674fcd43b61af911